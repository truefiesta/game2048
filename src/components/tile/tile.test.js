import React from "react";
import renderer from "react-test-renderer";
import Tile from "./tile";

describe(`Tile snapshot`, () => {
    it(`renders with 0`, () => {
        const tile = renderer.create(
            <Tile
                value={0}
            />
        )
        .toJSON();

        expect(tile).toMatchSnapshot();
    });

    it(`renders with 2048`, () => {
        const tile = renderer.create(
            <Tile
                value={2048}
            />
        )
        .toJSON();

        expect(tile).toMatchSnapshot();
    });
});
