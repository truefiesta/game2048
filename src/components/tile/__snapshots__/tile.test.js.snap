// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile snapshot renders with 0 1`] = `
<div
  className="tile tile-0"
>
  <span>
    0
  </span>
</div>
`;

exports[`Tile snapshot renders with 2048 1`] = `
<div
  className="tile tile-2048"
>
  <span>
    2048
  </span>
</div>
`;
